docs/Basics.md

```diff
@@ -15,10 +15,34 @@ Running Exoframe server on its own also doesn't require too much resources:
 - Exoframe Server consumes ~50mb of RAM
 - Traefik started along with server consumes ~60mb of RAM
 
-Be aware though - execution of deployments will result in (1) new Docker images being built and (2) new Docker containers being started.  
+Be aware though - execution of deployments will result in:
```

Comment:
I don't think the list is long enough to warrant this split. Please revert this.

Reply:
The list above it is literally two items too

Reply:
Sure, but that's a pretty common way of writing short lists. I don't see a problem with that.